formats/step_by_step_nav.jsonnet

```diff
@@ -0,0 +1,33 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: "step_by_step_nav",
+  publishing_app: "required",
+  links: {
+  },
+  definitions: (import "shared/definitions/_step_by_step_nav.jsonnet") + {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      required: [
+        "step_by_step_nav",
+      ],
+      properties: {
+        step_by_step_nav: {
```

Comment:
SInce you've already got a bunch of related definitions in _step_by_step_nav.jsonnet you might want to move this one in as well (as I guess you'd need that definition if you were reusing this stuff anyway)

Reply:
we couldn't make it work, so we going to leave it here for the time being.

```
(import "shared/default_format.jsonnet") + {
  document_type: "step_by_step_nav",
  publishing_app: "required",
  links: {
  },
  definitions: (import "shared/definitions/_step_by_step_nav.jsonnet") + {
    details: {
      type: "object",
      additionalProperties: false,
      required: [
        "step_by_step_nav",
      ],
      properties: {
        "$ref": "#/definitions/step_by_step_nav"
    }
  }
}

```

Reply:
I think it would be 

```
(import "shared/default_format.jsonnet") + {
  document_type: "step_by_step_nav",
  publishing_app: "required",
  links: {
  },
  definitions: (import "shared/definitions/_step_by_step_nav.jsonnet") + {
    details: {
      type: "object",
      additionalProperties: false,
      required: [
        "step_by_step_nav",
      ],
      properties: {
        step_by_step_nav: {
          description: "Some description",
          "$ref": "#/definitions/step_by_step_nav"
        }
      }
    }
  }
}
```

Reply:
That worked, thank you!

Reply:
Great 👍